Assignment2/question1.js

```diff
@@ -0,0 +1,33 @@
+/*
+    Declare a variable - let a;. On another line assign any value you like to a. Log the value of a before and after assignment. Try doing the same with const.
+    
+    Use the typeof operator to find the types of different variables. Specially note what the typeof operator returns for arrays, null values and NaN. How can you find if a variable is an array or NaN besides typeof?
+*/
+
+
+// let 
+let a;
+console.log(a) // gives undefined
+a = 100;
+console.log(a)
+
+// const
+const b = 200; // const if not initialized gives error
+console.log(b)
+console.log()
+
+// typeof
```

Comment:
write outputs of all typeof's in comments infront of each console log statement

Reply:
done

Reply:
added output